christfellowshipchurch/web-app
# [Chat] Query and use channelType from API

# Related Issue
- Depends on: **🛰️ api** > [[Chat] Add channelType to StreamChatType schema #158](https://github.com/christfellowshipchurch/apollos-api/pull/158/files)
- Relates to: **📱 app** > [[Chat] Query and use channelType from API #187](https://github.com/christfellowshipchurch/apollos-app/pull/187)

# Changes or Updates
- Updates queries for `LiveStream` and `GroupItem` to get `streamChatChannel.channelType` from the API
- Use that value ☝️ to initialize event and/or group chat contexts

# Test Instructions
- With `sandbox` mode enabled, go to a group's Chat
- Verify you can send/receive messages and that it is consistent/the same channel as what you see using the app on it's `channel-type-schema` branch

# Tests
